website/content/docs/customization.md

```diff
@@ -54,13 +54,15 @@ body {
 
 ## `registerPreviewTemplate`
 
-Registers a template for a collection.
+Registers a template for a folder collection or an individual file in a file collection.
 
-`CMS.registerPreviewTemplate(collection, react_component);`
+`CMS.registerPreviewTemplate(name, react_component);`
 
 **Params:**
 
-* collection: The name of the collection which this preview component will be used for.
+* name: The name of the entry which this preview component will be used for.
```

Comment:
> The name of the entry ...

I'm not so sure we should use the word "entry" here, as that is typically used to refer to a specific "file" in the docs. Any other ideas?